### ISSUE TITLE
training dataset format

### ISSUE THREAD

**Issue by zhebawomenjiuyaochangshenle** [MohsenZand/ObjectBox#29]
Hello, I always show "Can not train without labels." when I set the training set, how is the data in VOC format trained?I would be grateful if you could give me an example

**Comment by yangyang567** [comment]
maybe use COCO format,it work
